packages/@lwc/engine-server/src/__tests__/fixtures/wire/get-set/modules/x/wire/wire.js

```diff
@@ -0,0 +1,41 @@
+import { LightningElement, wire } from 'lwc';
+
+import { adapter } from './adapter';
+
+export default class Wire extends LightningElement {
+    @wire(adapter, { name: 'getterOnly' })
+    get getterOnly() {
+        throw new Error('get getterOnly should not be called');
```

Comment:
This component would be a good test case for `integration-karma` as well. `engine-dom` presumably works the same way.